src/electionguard/ballot.cpp

```diff
@@ -59,8 +61,26 @@ namespace electionguard
 
     // Property Getters
 
-    uint64_t PlaintextBallotSelection::toInt() const { return stoul(pimpl->vote); }
     string PlaintextBallotSelection::getObjectId() const { return pimpl->object_id; }
+    uint64_t PlaintextBallotSelection::getVote() const { return pimpl->vote; }
+    bool PlaintextBallotSelection::getIsPlaceholder() const
+    {
+        return pimpl->isPlaceholderSelection;
+    }
+
+    bool PlaintextBallotSelection::isValid(const std::string &expectedObjectId) const
```

Comment:
important to note that the constructor allows a bad value in but the isValid check fails for any value other than 0 or 1 currently.  this is to enable better future support for other vote types without requiring a change to the public API